src/app/legacy/providers/redeem-provider.tsx

```diff
@@ -74,6 +75,49 @@ export function RedeemProvider(props: { children: any }) {
     [inputToken, inputValue],
   )
 
+  const { data: quoteResult, isFetching: isFetchingQuote } = useQuery({
+    initialData,
+    queryKey: [
+      queryKey,
+      {
+        address,
+        inputToken,
+        inputValue,
```

Comment:
Usually, `inputTokenAmount` was the dependency here but it complains that it can't serialize BigInt's. Maybe you know a better way for this. It works but currently kind of repeating code to generate the input token amount below (see next comment)

Reply:
you can serialize the Bigint yourself basically i think, like just toString it